signingscript/test/test_utils.py

```diff
@@ -1,5 +1,72 @@
-from signingscript.utils import get_detached_signatures
+import os
+import tempfile
+import json
 
+import unittest
+from unittest.mock import patch
 
-def test_detached_signatures():
-    assert get_detached_signatures(["mar", "gpg", "pgp"]) == [("gpg", ".asc", "text/plain")]
+import asyncio
+import asynctest
+
+from signingscript.utils import get_detached_signatures, mkdir, load_json, raise_future_exceptions
+
+
+class UtilsTest(unittest.TestCase):
+    def test_detached_signatures(self):
+        self.assertEqual(get_detached_signatures(["mar", "gpg", "pgp"]), [("gpg", ".asc", "text/plain")])
+
+    def test_mkdir_does_make_dirs(self):
+        with tempfile.TemporaryDirectory() as test_dir:
+            end_dir = os.path.join(test_dir, 'dir_in_the_middle', 'leaf_dir')
+            mkdir(end_dir)
+
+            middle_dirs = list(os.scandir(test_dir))
+            self.assertDirIsUniqueAndNamed(middle_dirs, 'dir_in_the_middle')
+
+            leaf_dirs = list(os.scandir(middle_dirs[0].path))
+            self.assertDirIsUniqueAndNamed(leaf_dirs, 'leaf_dir')
```

Comment:
I'm curious what these other tests give you... just testing to make sure mkdir doesn't create extra files or dirs?